src/api/kube.ts

```diff
@@ -1859,12 +1859,20 @@ export class KubeHelper {
     }
   }
 
-  async clusterIssuerExists(name: string): Promise<boolean> {
+  /**
+   * Returns CRD version of Cert Manager
+   */
+  async getCertManagerK8sApiVersion(): Promise<string> {
+    const certManagerCRD = await this.getCrd('certificates.cert-manager.io')
+    return certManagerCRD.spec.version ? certManagerCRD.spec.version : CERT_MANAGER_CRD_DEFAULT_VERSION
+  }
+
+  async clusterIssuerExists(name: string, crdVersion = CERT_MANAGER_CRD_DEFAULT_VERSION): Promise<boolean> {
```

Comment:
`clusterIssuerExists` should not accept any versions.

```suggestion
  async clusterIssuerExists(name: string): Promise<boolean> {
  const version := async getCertManagerK8sApiVersion()
```

Reply:
Disagree. We can have different versions and need only that, which match currently used Cert Manager version.